### PR TITLE
Remove PlanningTestHelper from plan-consumer-test.ts

### PR DESCRIPTION
**Pull request by mykmartin**

### CODE REVIEW

**Comment by mmandlis** [comment]
argh https://github.com/PolymerLabs/arcs/pull/4123
we need to coordinate better :(
PS: i'll review tomorrow.

**Comment by mykmartin** [comment]
Argh^2. I almost didn't continue with this CL, but since it was close to
done just finished it up. I can change to a different task if you're happy
to plough through these, or you can let me know which files you aren't
working on.

File: src/planning/plan/tests/plan-consumer-test.ts [hunk 1]
```diff
@@ -10,22 +10,59 @@
...
+  Planner.clearCache();
+  const loader = new Loader();
+  const context = await Manifest.parse(manifestString, {loader, fileName: ''});
+  const arc = new Arc({
+    id: Id.fromString('demo'),
+    slotComposer,
+    loader,
+    context,
+    storageKey: 'volatile://!158405822139616:demo^^volatile-0',
+    inspectorFactory: devtoolsArcInspectorFactory
```
Comment (mmandlis) [inline src/planning/plan/tests/plan-consumer-test.ts]: i don't think you need devtools for this test, and it's easier to use the Runtime object to create arcs (see other examples)

File: src/planning/plan/tests/plan-consumer-test.ts [hunk 2]
```diff
@@ -10,22 +10,59 @@
...
+  });
+  const recipeIndex = RecipeIndex.create(arc);
```
Comment (mmandlis) [inline src/planning/plan/tests/plan-consumer-test.ts]: recipeIndex isn't needed for this test either. there is a helper method: `StrategyTestHelper.createTestStrategyArgs` you could use to set the planner init parameters

File: src/planning/plan/tests/plan-consumer-test.ts [hunk 3]
```diff
@@ -72,7 +107,7 @@ recipe
       assert.isEmpty(consumer.getCurrentSuggestions());
 
       // Updates suggestions.
-      await storeResults(consumer, helper.findSuggestionByParticleNames(['ItemMultiplexer', 'List']));
+      await storeResults(consumer, findSuggestionByParticleNames(suggestions, ['ItemMultiplexer', 'List']));
```
Comment (mmandlis) [inline src/planning/plan/tests/plan-consumer-test.ts]: in this particular case, there is only one suggestions, so it's easier to just verify the suggestions.length and the suggestions[0]'s particle names, no need for the helper method.

**Comment by mykmartin** [comment]
Thanks for reviewing, but I'll drop this in favour of yours. Sorry for this, I should have communicated that I was still working on the PR.

**Pull request closed without merging**